src/buffer_manager.cc

```diff
@@ -105,4 +105,29 @@ void BufferManager::clear_buffer_trash()
     m_buffer_trash.clear();
 }
 
+void BufferManager::arrange_buffers(ConstArrayView<String> first_ones)
+{
+    Vector<size_t> indices;
+    for (const auto& name : first_ones)
+    {
+        auto it = find_if(m_buffers, [&](auto& buf) { return buf->name() == name or buf->display_name() == name; });
```

Comment:
I think you can just compare the display name here.

Reply:
Yes that should be sufficient, and `buflist` contains the display names, but on the other hand it's not like there is any risk of ambiguity when comparing the full name either.